core/data-source/api/src/main/java/com/timers/stopwatch/core/data/source/api/DataSource.kt

```diff
@@ -0,0 +1,23 @@
+package com.timers.stopwatch.core.data.source.api
```

Comment:
api module should not contain logic (default values), it should just contain interfaces of DataSources